lib/package/commands.coffee

```diff
@@ -22,6 +22,14 @@ module.exports =
         @withInk ->
           boot()
           juno.runtime.evaluation.eval(move: true)
+      'julia-client:run-cell': =>
+        @withInk ->
+          boot()
+          juno.runtime.evaluation.eval(cell: true)
+      'julia-client:run-cell-and-move': =>
+        @withInk ->
+          boot()
+          juno.runtime.evaluation.eval(cell: true, move: true)
```

Comment:
Thanks for the review. Will fix up the rest tomorrow. Just quickly on this point, it does move to the start of the next cell, because, as I was pleased to discover, `blocks.moveNext` goes to the end of the `range` provided, which is the end of the cell in this case.